lib/itests/core/src/test/resources/mappings/atlasmapping-multiplicity-transformation-action_repeat.json

```diff
@@ -0,0 +1,76 @@
+{
+  "AtlasMapping": {
+    "jsonType": "io.atlasmap.v2.AtlasMapping",
+    "dataSource": [
+      {
+        "jsonType": "io.atlasmap.json.v2.JsonDataSource",
+        "id": "json-source-repeat",
+        "uri": "atlas:json:json-source-repeat",
+        "dataSourceType": "SOURCE"
+      },
+      {
+        "jsonType": "io.atlasmap.json.v2.JsonDataSource",
+        "id": "json-target",
+        "uri": "atlas:json:json-target",
+        "dataSourceType": "TARGET"
+      }
+    ],
+    "mappings": {
+      "mapping": [
+        {
+          "jsonType": "io.atlasmap.v2.Mapping",
+          "inputFieldGroup": {
+            "jsonType": "io.atlasmap.v2.FieldGroup",
+            "actions": [
+              {
+                "Expression": {
+                  "expression": "Repeat(count(${1}), ${0})"
```

Comment:
Like we talked in the meeting, let's test with new expression instead of this deprecated field action expression:
https://github.com/atlasmap/atlasmap/blob/master/lib/itests/core/src/test/resources/mappings/atlasmapping-expression-filter-select-java.json#L18

Reply:
@rkorytkowski  - Modified to new expression on 2 mapping files. please take the new files from below links and commit to the PR

https://github.com/shanthi4647/atlasmap/blob/2276/lib/itests/core/src/test/resources/mappings/atlasmapping-multiplicity-transformation-action_repeat.json

https://github.com/shanthi4647/atlasmap/blob/2276/lib/itests/core/src/test/resources/mappings/atlasmapping-multiplicity-transformation-action_repeat_nested_collection.json

Reply:
@shanthi4647 I'm sorry for responding late. I was away for a while. I've applied your fixes. Thanks!